tests/integration/test_wpk/test_wpk_manager_task_states.py

```diff
@@ -202,6 +270,42 @@ def overwrite_node_name(value):
 
 def test_wpk_manager_task_states(get_configuration, configure_environment,
                                  restart_service, configure_agents):
+    '''
+    description: Register agents in the manager and test upgrading
+                 them through a command, which will make the agent
+                 to upgrade using WPK packages.
```

Comment:
```suggestion
    description: Agents can be upgraded remotely. This upgrade is performed by the manager which sends each registered agent a WPK (Wazuh signed package) file that contains the files needed to upgrade the agent to the new version. These tests ensure, the behaviour of the WPK upgrade on the manager side, in case of the manager stopped before finishing the upgrade.
```

Reply:
Done in this [commit](https://github.com/wazuh/wazuh-qa/pull/2409/commits/210acc6c36122300661af5bbcefdc5b6d158bbb1)